js/Common.js

```diff
@@ -1742,8 +1742,28 @@ if (!CRM.vars) CRM.vars = {};
         }
         $(this).parent().toggleClass('collapsed');
         e.preventDefault();
+      })
+
+      // Save the state for sticky accordions
+      .on('click', 'details.crm-accordion-sticky', function(e) {
+        // Workaround to run last, otherwise the open attribute is not yet updated
+        setTimeout(() => {
+          CRM.cache.set(this.id, document.getElementById(this.id).hasAttribute('open'));
+        }, 0);
       });
 
+    // Expand sticky accordions
+    Array.from(document.querySelectorAll('.crm-container details.crm-accordion-sticky')).forEach((expander) => {
+      var state = CRM.cache.get(expander.id);
+      if (state === true) {
+        document.getElementById(expander.id).setAttribute('open', '');
+      }
+      else if (state === false) {
+        // An accordion might be open by default, and we want to close it
+        document.getElementById(expander.id).removeAttribute('open');
+      }
```

Comment:
could be simplified?

```suggestion
      if (state === true || state === false) {
        expander.toggleAttribute('open', state);
      }
```